base/db/R/insert.format.vars.R

```diff
@@ -39,44 +39,44 @@
 #'   formats_variables = formats_variables_tibble)
 #' }
 insert.format.vars <- function(con, format_name, mimetype_id, notes = NULL, header = TRUE, skip = 0, formats_variables = NULL, suppress = TRUE){
-  
+
   # Test if name is a character string
   if(!is.character(format_name)){
     PEcAn.logger::logger.error(
       "Name must be a character string"
     )
   }
-  
+
   # Test if format name already exists
-  name_test <- dplyr::tbl(con, "formats") %>% dplyr::select(id, name) %>% dplyr::filter(name %in% !!format_name) %>% collect()
+  name_test <- dplyr::tbl(con, "formats") %>% dplyr::select(.data$id, .data$name) %>% dplyr::filter(.data$name %in% !!format_name) %>% .data$collect()
```

Comment:
`collect` is a function -- it doesn't need the `.data` pronoun.

```suggestion
  name_test <- dplyr::tbl(con, "formats") %>% dplyr::select(.data$id, .data$name) %>% dplyr::filter(.data$name %in% !!format_name) %>% collect()
```